README.md

```diff
@@ -256,6 +255,35 @@ docker run --rm -it --net=host -v ${PWD}:/nConnect/data nknorg/nconnect
 
 followed by the command line argument you want to add.
 
+## nConnect Client Connects to Multi Servers
+
+Now nConnect client can connect to multi servers. You can edit `config.json` to add multi servers admin address:
```

Comment:
Should we also add examples for cli argument?

Reply:
Yes. example is added.